src/renderer/viz/utils/featureVizProperties.js

```diff
@@ -0,0 +1,2 @@
+const FEATURE_VIZ_PROPERTIES = ['color', 'width', 'strokeColor', 'strokeWidth', 'symbol', 'symbolPlacement'];
```

Comment:
Some properties are missing here (at least `filter` and `transform`), I don't know if adding them will have unexpected consequences though

Reply:
Good catch. I have added them both and the tests pass